src/Sentry.Unity.Editor/SentryCli.cs

```diff
@@ -102,5 +102,21 @@ internal static void AddExecutableToXcodeProject(string projectPath, IDiagnostic
             File.Copy(executableSource, executableDestination);
             SetExecutePermission(executableDestination);
         }
+
+        internal static string? UrlOverride(string? dsnOption, string? urlOverrideOption)
```

Comment:
Because it may not be needed to specify the override in the first place. If you've configured the DSN to be an internal server that's available during build, it should just work without specifying the same server as upload override. As is, it would try to upload to sentry.io instead, if you didn't specify any override.